distribution/docs/src/main/jdocs/content/_configuring/admin-console-tutorial-contents.adoc

```diff
@@ -0,0 +1,113 @@
+:title: Admin Console Tutorial
+:type: configuringAdminConsole
+:status: published
+:summary: Tutorial for the ${admin-console}.
+:order: 01
+
+.${admin-console} Tutorial
+****
+This overview covers general uses for the ${admin-console}.
```

Comment:
❓ This section seems to have a background down to the end of the page at `Disabled apps will be moved to a list at the bottom of the`. Is that intended?